Apps/frontend/src/views/Waves.svelte

```diff
@@ -102,6 +102,17 @@
     webGLPlot.update();
     window.requestAnimationFrame(newFrame);
   };
+
+  export const resetPlot = () => {
+    xArr = new Array(NUM_CHANNELS).fill(0.0);
+    xLast = new Array(NUM_CHANNELS).fill(0);
+    channel_samples = Array.from(Array(NUM_CHANNELS), () =>
+      new Array(CANVAS_WIDTH).fill(0.0)
+    );
+    update();
+    webGLPlot.clear();
+    console.log("clear");
+  }
```

Comment:
You should run the linter and prettier over the code again. For me, it changes several files, including this line.